src/containers/login/Login.jsx

```diff
@@ -8,13 +6,7 @@ export default function Login() {
   
     return (
         <LoginTemplate title="Getting Social" subtitle="Nice to see you around here! 
-        Please log in first.">
+        Please log in first." leftLink={["Forgot password?" , "#"]} rightLink={["First time here? Sign in", "Signin"]}>
```

Comment:
leftLink | rightLink could be an object like: {label: "Forgot password?", to: "#"}